plugins/uefi-capsule/fu-uefi-backend-linux.c

```diff
@@ -70,15 +76,53 @@ fu_uefi_backend_device_new (const gchar *path)
 	return g_steal_pointer (&dev);
 }
 
+void
+fu_uefi_backend_set_enable_cod (FuBackend *backend, gboolean enable_cod)
+{
+	FuUefiBackend *self = FU_UEFI_BACKEND (backend);
+	self->enable_cod = !enable_cod;
```

Comment:
inverting a negative we get from the config file makes this confusing, can we just set the disable instead?

Reply:
whoops, that inversion was a typo.